### PR TITLE
chore: version bump pr template

### PR DESCRIPTION
**Pull request by andreabadesso**
### Acceptance Criteria
- We should have a default PR template with links to two PR templates -- the version bump and the feature branch PR templates
- The version bump PR template should have a checklist for when creating a version bump in the wallet-mobile

### Security Checklist
- [x] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Acceptance Criteria
+- Include here all things that this PR should solve
```
Comment (luislhl) [inline .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md:2]: Should we change this to something like: "Include links to all PRs that will be included in this version"?

File: .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+### Checklist
+- [ ] Make sure you updated the `CURRENT_PROJECT_VERSION` with the appropriate release-candidate version in `ios/HathorMobile.xcodeproj/project.pbxproj`
+- [ ] Make sure you updated the `MARKETING_VERSION` with the appropriate version in `ios/HathorMobile.xcodeproj/project.pbxproj`
+- [ ] Make sure you updated the version attribute in `package.json`
+- [ ] Make sure you updated the version attribute in `package-lock.json`
+- [ ] Make sure you incremented the `versionCode` attribute in `android/app/build.gradle`
+- [ ] Make sure you updated the `versionName` with the appropriate version, including the release candidate number in `android/app/build.gradle`
+
+### Security Checklist
+- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.
```
Comment (luislhl) [inline .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md:13]: thought: Maybe this is unnecessary in this template. What do you think?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,6 +1,4 @@
-### Acceptance Criteria
-- Include here all things that this PR should solve
+Please go the the `Preview` tab and select the appropriate Pull Request template:
 
-
-### Security Checklist
-- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.
+* [Feature Branch](?expand=1&template=feature_branch_pr_template.md) - Use this PR template when you are merging a feature branch into `dev`
+* [Version Bump](?expand=1&template=version_bump_pr_template.md) - Use this PR template when you are merging a version bump branch into `dev`
```
Comment (pedroferreira1) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: It's not exactly when you are merging a branch, it's when you are creating a pull request of a feature branch / version bump.

**Review by pedroferreira1 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md (hunk 2 above)
Comment (andreabadesso) [reply .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md]: I agree, done in 259bf8a64a533bedbfafbfc4033a4d42a4eebe70

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (andreabadesso) [reply .github/PULL_REQUEST_TEMPLATE.md]: I agree, done in aeb55c50473c481acf71bfa25284f3233ccbfde2

File: .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md (hunk 1 above)
Comment (andreabadesso) [reply .github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md]: I think we should, done in 6c728840a6320d744fcba7207ad68dae69edd281

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (andreabadesso) [reply .github/PULL_REQUEST_TEMPLATE.md]: Updated the text to be less repetitive in 6c728840a6320d744fcba7207ad68dae69edd281

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**